kythe/typescript/indexer.ts

```diff
@@ -977,6 +979,10 @@ class Visitor {
         this.emitNode(vname, 'variable');
 
         this.emitEdge(this.newAnchor(decl.name), 'defines/binding', vname);
+
+        if (ts.isComputedPropertyName(decl.name)) {
+          this.visit(decl.name.expression);
+        }
```

Comment:
Maybe we should just recurse with ts.forEachChild?  Not sure.

Reply:
Done.